.gitignore

```diff
@@ -4,3 +4,12 @@ docs/content/learn/changelog.adoc
 docs/static/download/*.zip
 docs/public/
 docs/resources/
+
+# CMake
```

Comment:
I don't want to add user specific stuff to .gitignore. Those belong in your user global .gitignore or .git/ignore. The project .gitignore contains only files that are generated by the project's build system.

Reply:
I got it.